packages/components/src/IconList/src/IconList.tsx

```diff
@@ -0,0 +1,68 @@
+import { Provider, useContextProps } from "react-aria-components";
```

Comment:
To ensure a certain consistency in imports, I suggest using this rule as a guide. 

https://gsoftdev.atlassian.net/wiki/spaces/TEC/pages/3686760494/Coding+Standards#Import-order

Reply:
Can this be enforced via eslint?

Reply:
It's a good idea to ask the question in this more needy way.

Reply:
Logged this as a separate issue: https://github.com/gsoft-inc/wl-hopper/issues/178